signer/src/message.rs

```diff
@@ -124,6 +128,120 @@ impl From<WstsMessage> for Payload {
     }
 }
 
+impl From<SweepTransactionInfo> for Payload {
+    fn from(value: SweepTransactionInfo) -> Self {
+        Self::SweepTransactionInfo(value)
+    }
+}
+
+/// Represents information about a new sweep transaction.
+#[derive(Debug, Clone, PartialEq, serde::Serialize, serde::Deserialize)]
+pub struct SweepTransactionInfo {
```

Comment:
We might want to include the actual transaction here as well.

Reply:
I'm not a huge fan of throwing around external types in a proprietary messaging protocol tbh. The signer can retrieve the transaction from their db/node if they want it?

Reply:
Yeah, I think that you're right, including the transaction is not necessary here.